README.md

```diff
@@ -157,67 +173,76 @@ TBD
     - TiNDA will stop if there are less than 50 rare germline variants
 
-* Version update to 1.3.0
+* Version update to 1.3.0 ([Column bug](#column-bug))
 
   - Added `tumorSample` and `controlSample` variables to job environments. These can also be used in output file paths.
 
-* Version update to 1.2.178
+* Version update to 1.2.178 ([Column bug](#column-bug))
 
   - Including gnomAD exomes and genomes for nocontrol workflow filtering.
   - Adding gnomAD files for no-control workflow.
-  - Updating the COWorkflowsBasePlugin.
-  - Including gnomAD exomes and genomes for nocontrol workflow filtering
+  - Updated from COWorkflowsBasePlugin:1.0.3 to COWorkflowsBasePlugin:1.1.0.
 
-* Version update to 1.2.177-8
+* Version update to 1.2.177-8 (=1.2.182) ([Column bug](#column-bug))
 
-* Version update to 1.2.177-7
+  - major: Changed `GERMLINE_AVAILABLE` with allowed values 0 and 1 to `isNoControlWorkflow` with allowed values "false" and "true".
+  - minor: Update to from COWorkflows:1.2.59 to COWorkflowsBasePlugin:1.0.3.
+  - minor: Added writing TiN classification to the VCF file.
+  - Note that the version may report itself as 1.2.182 according to the `versionInfo.txt`.
 
-* Version update to 1.2.177-601 (ReleaseBranch_1.2.177-6)
-
-  - Bugfix: Fix of column swap bug introduced before 1.0.167. Output VCF with swapped control and tumor genotype columns if they are in the 11th and 10th column respectively.
+* Version update to 1.2.177-7 ([Column bug](#column-bug))
 
-* Version update to 1.2.177-6
+  - patch: Fix problem with dealing with empty result files in `indelCalling.sh`.
+  
+* Version update to 1.2.177-6 ([Column bug](#column-bug))
 
-  - removed a `umask` from `checkSampleSwap_TiN.sh`
+  - Removed a `umask` from `checkSampleSwap_TiN.sh`.
 
-* Version update to 1.2.177-5
+* Version update to 1.2.177-5 ([Column bug](#column-bug))
 
   - Platypus update 0.8.1 to 0.8.1.1 (this only fixes the reported Platypus version number in the VCF)
   - Added conda environment
   - Reordered cvalues in plugin XML configuration
   - Neutral refactoring in `indelCalling.sh`
   - Bugfix: JSON syntax for "file"
 
-* Version update to 1.2.177-4 (includes development versions 1.2.177-1 to -3)
+* Version update to 1.2.177-4 (includes development versions 1.2.177-1 to -3) ([Column bug](#column-bug))
 
   - Roddy 3.0 support
   - Updated COWorkflows base plugin from 1.2.59 to 1.2.76
   - Fixed reported version
   - Bugfix in `indel_extractor_v1.pl` concerning ncRNA_exonic and ncRNA_splicing annotations for somatic calls.
-  - Added `runTindo` cvalue
+  - Added `runTinda` cvalue
 
-* Version update to 1.0.176-10 (=1.2.177-1)
+* Version update to 1.0.177-2 ([Column bug](#column-bug))
+
+  - Bugfix ncRNA file
+  - Allow toggling on/off of Tinda
+
+* Version update to 1.0.176-10 (=1.0.177-1) ([Column bug](#column-bug))
 
   - Added right and bottom border features to `TiN_canopyBasedClustering.R`
   - Version is not backwards compatible
```

Comment:
Updates were almost on the TiNDA part (1.0.176-10 vs 1.0.176-9):
- Major: Added a method based on maximum `Control_AF` to remove unusual cluster.
- Major: Increased the number of iterations to 100 (from 50) in the EM step. 
- - TiNDA rescue clusters will not be same
- Minor: Added a polygon shape to the TiNDA plot
- - Plots will look different
- Patch: Parametrised the right and left threshold that defines the border for TiNDA cluster selection.
- - Uses the same values as before
- Patch: Fixed the bugs in the bash sanity checks.

Reply:
Perfect! Thanks!